oidc_client_example/index_tmp.html

```diff
@@ -17,10 +17,22 @@
     <!-- Use JQuery -->
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
     <!-- Use JS OpenID library -->
-    <script src="https://rawgit.com/IdentityModel/oidc-client-js/dev/dist/oidc-client.min.js"></script>
+    <script src="https://cdn.rawgit.com/IdentityModel/oidc-client-js/release/dist/oidc-client.min.js"></script>
 
     <script type="text/javascript">
     $(function() {
+
+        function createRandomString(length) {
+            var alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+            var result = "";
+
+            for (var i = 0; i < length; i++)
+            {
+                result += alphabet[Math.floor(Math.random() * alphabet.length)];

```

Comment:
`Math.random()` [should not be used for cryptographic purposes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random)